Converters/Rhino/Speckle.Converters.RhinoShared/ToSpeckle/Raw/BrepToSpeckleConverter.cs

```diff
@@ -43,7 +43,7 @@ IConverterSettingsStore<RhinoConversionSettings> settingsStore
   public SOG.Brep Convert(RG.Brep target)
   {
     var tol = _settingsStore.Current.Document.ModelAbsoluteTolerance;
-    target.Repair(tol);
+    //target.Repair(tol);
```

Comment:
to not forget later, would be nice if you create ticket to have it as setting if it make sense? i don't remember what we agreed on before

Reply:
uncommented and added note on why it's potentially tricky